server/views/pages/games/anagramica.html

```diff
@@ -1,194 +1,168 @@
-{% from "../../components/feedback-widget/macro.njk" import hubFeedbackWidget %}
 {% from "govuk/components/radios/macro.njk" import govukRadios %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
-{% from "../../components/user-details/macro.njk" import userDetails %}
-{% extends "../../components/template.njk" %}
-
-
-{% block pageTitle %}
-  {{ title }}
-{% endblock %}
+{% extends "../../components/basicTemplate.njk" %}
 
 {% block head %}
 	<link href="/public/javascript/games/anagramica/stylesheets/style.css" rel="stylesheet"/>
-  <link href="/public/stylesheets/application.css" rel="stylesheet"/>
+  {{ super() }}
```

Comment:
Sorry, not sure where `super()` was called and what it does

Reply:
https://mozilla.github.io/nunjucks/templating.html#super
pulls in the parent blocks contents